README.md

```diff
@@ -2,79 +2,79 @@
+To submit your text, we ask that you create a GitHub account. When you are ready to submit your written documentation for approval, follow these easy steps:
 
 ### With the GitHub GUI
 
-Most tasks can be completed from the web GUI on GitHub. Here's an example of adding a file you've created on your local machine to the Rocky Linux documentation GitHub repository.
+You can complete almost all tasks from the web GUI on GitHub. Here's an example of adding a file you've created on your local machine to the Rocky Linux documentation GitHub repository.
 
-1. Login to your GitHub account.
-2. Navigate to the Rocky Linux Documentation repository at: [https://github.com/rocky-linux/documentation.git](https://github.com/rocky-linux/documentation.git)
-3. You should be on the "Main" branch, so check the drop down label down in the middle section, just to be sure you are. Your document might not ultimately end up in the "Main" branch, but admins will move it around to where it logically fits later.  
-On the right-hand side of the page, click the "Fork" button, which will create your own copy of the documentation.
-4. In the middle of the page on the forked copy, just to the left of the Green "Code" drop down, is an "Add file" button. Click this and choose the "Upload files" option.
-5. This will give you a way to either drag and drop files here, or to browse to them on your computer. Go ahead and use the method you are most comfortable with.
-6. Once the file is uploaded, the next thing you need to do is create a Pull Request. This request lets the upstream administrators know that you've got a new file (or files) that you would like them to merge with the master.  
+1. log in to your GitHub account.
+2. Navigate to the Rocky Linux Documentation repository at [https://github.com/rocky-linux/documentation.git](https://github.com/rocky-linux/documentation.git)
+3. You should be on the "Main" branch, so check the drop-down label down in the middle section to be sure you are. Your document might not ultimately end up in the "Main" branch, but admins will move it around to where it logically fits later.  
+On the right-hand side of the page, click the "Fork" button, which will create your copy of the documentation.
+4. In the middle of the page on the forked copy, just to the left of the Green "Code" drop-down, is an "Add file" button. Click this and choose the "Upload files" option.
+5. This will give you a way to either drag and drop files here or browse to them on your computer. Go ahead and use the method which you prefer.
+6. Once the file is uploaded, the next thing you need to do is create a Pull Request. This request lets the upstream administrators know that you have a new file (or files) that you would like them to merge with the master.  
 Click on "Pull Request" in the upper-left of the screen.
```

Comment:
(New document, revision, suggested change, etc.,) should be (New document, revision, suggested change, etc.), I missed this